pkg/plugins/runtime/k8s/controllers/pod_converter_test.go

```diff
@@ -250,6 +245,11 @@ var _ = Describe("PodToDataplane(..)", func() {
 			servicesForPod: "18.services-for-pod.yaml",
 			dataplane:      "18.dataplane.yaml",
 		}),
+		Entry("19. Terminating pod is unhealthy", testCase{
```

Comment:
Obviously not super important or even something for this PR but numbers in the test output seem superfluous/noisy

Reply:
I kind of agree. I can make this change in separate PR if you want

Reply:
Was just a note for next time I suppose, this probably isn't the only place it's done. Very low priority